test/endpoints/automations.test.js

```diff
@@ -1,15 +1,35 @@
 import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import axios from 'axios';
 import chai, { expect } from 'chai';
+import * as resource from '../../server/modules/allocations';
+import automationData from '../mocks/automationData';
 import app from '../../server';
 
 chai.use(chaiHttp);
 
 describe('Tests for automation endpoints\n', () => {
-  it('Should return automation data with a 200 response code', (done) => {
+  before(async () => {
+    sinon.stub(axios, 'get').callsFake(
+      () => new Promise((resolve) => {
+        resolve(automationData);
+      }),
+    );
+    sinon.stub(Date, 'now').callsFake(() => 1542376484108);
+    const result = await resource.fetchNewPlacements('Placed - Awaiting Onboarding', 3);
+    console.log('====================================');
+    console.log(result);
+    console.log('====================================');
+  });
+
```

Comment:
@janetnim 
```suggestion
describe('Tests for automation endpoints\n', () => {
  var axiosStub, dateStub;
  before(async () => {
    axiosStub = sinon.stub(axios, 'get').callsFake(
      () => new Promise((resolve) => {
        resolve(automationData);
      }),
    );
    dateStub = sinon.stub(Date, 'now').callsFake(() => 1542376484108);
    const result = await resource.fetchNewPlacements('Placed - Awaiting Onboarding', 3);
    console.log('====================================');
    console.log(result);
    console.log('====================================');
  });
  after(()=> {
    axiosStub.restore();
    dateStub.restore();
  })
```
Always restore the stubbed object to prevent it from affecting other test stubs for example the error in circle ci
```bash
2) Partner and Allocations Test Suite
       Should fetch new placements based on the status specified, and the numberOfDays provided:
     TypeError: Attempted to wrap get which is already wrapped
      at checkWrappedMethod (node_modules/sinon/lib/sinon/util/core/wrap-method.js:38:21)
      at wrapMethod (node_modules/sinon/lib/sinon/util/core/wrap-method.js:85:13)
      at stub (node_modules/sinon/lib/sinon/stub.js:71:44)
      at Sandbox.stub (node_modules/sinon/lib/sinon/sandbox.js:291:33)
      at Context.stub (test/modules/allocations.test.js:11:32)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:288:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:114:21)
      at asyncGeneratorStep (test/modules/allocations.test.js:21:103)
      at _next (test/modules/allocations.test.js:23:194)
      at /home/circleci/repo/test/modules/allocations.test.js:23:364
      at new Promise (<anonymous>)
      at Context.<anonymous> (test/modules/allocations.test.js:23:97)
  --------------
  Error: Stack Trace for original
      at wrapMethod (node_modules/sinon/lib/sinon/util/core/wrap-method.js:110:31)
      at stub (node_modules/sinon/lib/sinon/stub.js:71:44)
      at Sandbox.stub (node_modules/sinon/lib/sinon/sandbox.js:291:33)
      at Context.stub (test/endpoints/automations.test.js:13:11)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:288:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:114:21)
      at asyncGeneratorStep (test/endpoints/automations.test.js:21:103)
      at _next (test/endpoints/automations.test.js:23:194)
      at /home/circleci/repo/test/endpoints/automations.test.js:23:364
      at new Promise (<anonymous>)
      at Context.<anonymous> (test/endpoints/automations.test.js:23:97)

  3) Partner and Allocations Test Suite
       Should throw an error when no partner record was found:
     TypeError: Attempted to wrap get which is already wrapped
      at checkWrappedMethod (node_modules/sinon/lib/sinon/util/core/wrap-method.js:38:21)
      at wrapMethod (node_modules/sinon/lib/sinon/util/core/wrap-method.js:85:13)
      at stub (node_modules/sinon/lib/sinon/stub.js:71:44)
      at Sandbox.stub (node_modules/sinon/lib/sinon/sandbox.js:291:33)
      at Context.stub (test/modules/allocations.test.js:57:32)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:288:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:114:21)
      at asyncGeneratorStep (test/modules/allocations.test.js:21:103)
      at _next (test/modules/allocations.test.js:23:194)
      at /home/circleci/repo/test/modules/allocations.test.js:23:364
      at new Promise (<anonymous>)
      at Context.<anonymous> (test/modules/allocations.test.js:23:97)
  --------------
  Error: Stack Trace for original
      at wrapMethod (node_modules/sinon/lib/sinon/util/core/wrap-method.js:110:31)
      at stub (node_modules/sinon/lib/sinon/stub.js:71:44)
      at Sandbox.stub (node_modules/sinon/lib/sinon/sandbox.js:291:33)
      at Context.stub (test/endpoints/automations.test.js:13:11)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:288:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:114:21)
      at asyncGeneratorStep (test/endpoints/automations.test.js:21:103)
      at _next (test/endpoints/automations.test.js:23:194)
      at /home/circleci/repo/test/endpoints/automations.test.js:23:364
      at new Promise (<anonymous>)
      at Context.<anonymous> (test/endpoints/automations.test.js:23:97)

```

Reply:
Thanks for that. I'll implement the changes suggested